doc/source/development/maintaining.rst

```diff
@@ -463,23 +467,19 @@ which will be triggered when the tag is pushed.
    - Set as the latest release: Leave checked, unless releasing a patch release for an older version
      (e.g. releasing 1.4.5 after 1.5 has been released)
 
-5. The GitHub release will after some hours trigger an
+5. Upload wheels to PyPI::
+
+    twine upload pandas/dist/pandas-<version>*.{whl,tar.gz} --skip-existing
```

Comment:
Still not fond of using the pypi publish action https://github.com/pypa/gh-action-pypi-publish (by gating it behind a manual trigger)?

Reply:
Nice yeah I think that would work. I can also try to look into this sometime in the future

Reply:
The latest version of the GitHub action now also implements the "Trusted Publisher" model of PyPI (https://github.com/marketplace/actions/pypi-publish#trusted-publishing), avoiding creating an API token that is stored as a secret in github